Gemfile

```diff
@@ -52,6 +52,7 @@ gem "pg", ">= 0.18", "< 2.0"
 gem "premailer-rails", "1.10.3"
 gem "premailer", "1.11.1"
 gem "pretender", "~> 0.3.3"
+gem "progressbar", "~> 1.10.1"
```

Comment:
This library is kinda big. Could we use [tty-progressbar](https://github.com/piotrmurach/tty-progressbar) Or do require false here since you have an explicit require in the task?

Reply:
Good call. I'll use `require: false`